src/Compilers/CSharp/Test/Syntax/LexicalAndXml/XmlDocCommentTests.cs

```diff
@@ -3269,6 +3269,48 @@ public void WriteDocumentationCommentXml_01()
                 );
         }
 
+        [WorkItem(47278, "https://github.com/dotnet/roslyn/issues/47278")]
+        [Fact]
+        public void NormalizeWhitespace()
+        {
+            var text = @"namespace Foo
+{
+    /// <summary>
+    /// This is Foo.Bar
```

Comment:
>Foo [](start = 16, length = 3)

"Foo" is getting flagged by policheck. Please remove usages of this word. from the tests.